v2/datastream-to-spanner/terraform/samples/pre-configured-conn-profiles/terraform.tfvars

```diff
@@ -0,0 +1,88 @@
+# Common Parameters
+common_params = {
+  project                         = "<YOUR_PROJECT_ID>"
+  region                          = "<YOUR_GCP_REGION>"
+  migration_id                    = "<YOUR_MIGRATION_ID>" #Will be used as a prefix for all resources, auto-generated if not specified
+  add_policies_to_service_account = "<TRUE/FALSE>"        # This will decide if roles will be attached to service accounts or not.
+}
+
+# Datastream Parameters
+datastream_params = {
+  source_connection_profile_id  = "<YOUR_SOURCE_CONNECTION_PROFILE_ID>"
+  target_connection_profile_id  = "<YOUR_TARGET_CONNECTION_PROFILE_ID>"
+  target_gcs_bucket_name        = "<YOUR_GCS_BUCKET_NAME>"
+  pubsub_topic_name             = "live-migration" # Or your custom topic name
+  stream_id                     = "mysql-stream"   # Or your custom stream ID
+  max_concurrent_cdc_tasks      = 50               # Adjust as needed
```

Comment:
I think this is 8 or so by default. Lets set sensible defaults here as folks probably won't change this unless required.

Reply:
I added 50 because that's what we use in SMT and wanted to keep the configurations same. 
Do you have a different default in mind? If we want to use Datastream defaults (20), we can just remove this from here.

Reply:
Resolved